CHANGELOG.md

```diff
@@ -34,6 +34,21 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
```

Comment:
i have a question about the three following PRs, which seems like bug fixes to me, and should go in 0.40.1:
- https://github.com/cosmos/cosmos-sdk/pull/8359
- #8300 
- #8301

Reply:
#8359 needs to be backported. The relayer won't work without it. I forgot to add a changelog entry, I guess I can open a new pr to master with the entry and then that should be cherry picked as well?

Reply:
Would be great if we could backport #8341 as well